ipa-core/src/protocol/hybrid/agg.rs

```diff
@@ -187,6 +187,7 @@ pub mod test {
     // the inputs are laid out to work with exactly 2 shards
     // as if it we're resharded by match_key/prf
     const SHARDS: usize = 2;
+    const SECOND_SHARD: ShardIndex = ShardIndex::from_u32(1);
```

Comment:
shouldn't this just be `from`?

```suggestion
    const SECOND_SHARD: ShardIndex = ShardIndex::from(1);
```

Reply:
trait methods [are not supported](https://github.com/rust-lang/rust/issues/67792) in const contexts yet